SPECS/lua/lua-5.4.2-shared_library-1.patch

```diff
@@ -0,0 +1,78 @@
+Submitted By:            Douglas R. Reno <renodr at linuxfromscratch dot org>
+Date:                    2020-06-30
+Initial Package Version: 5.4.0
+Upstream Status:         Rejected
```

Comment:
What should we make of this rejected upstream status?

Reply:
As far as I could tell, upstream does not ship a shared library for Lua and has no desire to at this time.  I'd rather not remove this patch for backwards-compatibility purposes.